test_vm/vendor/cookbooks/dbfit_test/recipes/diskspace.rb

```diff
@@ -0,0 +1,17 @@
+chef_gem 'di-ruby-lvm'
+
+lvm_physical_volume '/dev/sdb' do
+  action :create
+end
+
+lvm_volume_group 'centos' do
+  action :extend
+  physical_volumes ['/dev/sdb']
+end
+
+lvm_logical_volume 'root' do
+  group 'centos'
+  action :resize
+  size '9G'
+  take_up_free_space true
```

Comment:
Without the size we get an error: -

> ==> default: [2015-09-13T18:36:12+00:00] ERROR: lvm_logical_volume[root] (dbfit_test::diskspace line 12) had an error: Chef::Exceptions::ValidationFailed: Required argument size is missing!

I was expecting that we'd only need to specify `take_up_free_space true` to use all of the new space.

If I don't specify `take_up_free_space true` then the file system doesn't report the extra space being available and the provisioning runs out of disk space.